infra-l2-outbound-proxy/infra-test/lib/test-helper.ts

```diff
@@ -0,0 +1,21 @@
+import { Template } from "aws-cdk-lib/assertions";
+import { readFileSync } from "fs";
+import { load } from "js-yaml";
+import { schema } from "yaml-cfn";
+
+export class testHelper {
+  private template: Template;
+
+  public getTemplate<T extends Template>() {
+    if (this.template === null || this.template === undefined) {
+      const yamltemplate: any = load(
+        readFileSync("../template.yaml", "utf-8"),
```

Comment:
Does it need to go 2 level up for template file?

Reply:
no I don't think so. Although it is in a `lib/` folder the npm script runs from `infra-test` hence `..` is fine